content/en/apps/guides/hosting/4.x/app-developer.md

```diff
@@ -128,18 +126,95 @@ This script brings a lot of benefits with it:
 * All compose files and images will be downloaded automatically for you
 * All networks, storage volumes and containers will be created 
 * A valid TLS certificate will be installed, allowing you to easily test on with CHT Android natively on a mobile device
+* An unused port is automatically chosen for you when creating a new project.  No more manually looking at your existing `.env` files!
 
-Currently, the CHT Docker Helper for 4.0 is in beta, and comes with some caveats:
-* it is not yet available in `master`
-* it does not account for port conflicts, so you need to manually resolve them yourself by editing the `*.env` files
-* upgrades are not yet supported
-* concurrently running multiple instances is not yet supported
+### Installing
 
 To get started using it:
 1. Clone the [CHT Core](https://github.com/medic/cht-core/) repo
-2. Checkout the `7852-cht-4.x-docker-helper` branch
-3. `cd` into the `cht-core/scripts/docker-helper-4.x/` directory
-4. Run the script with `./cht-docker-compose.sh`
+4. When you want to check for updates, just run `git pull orign` in the `cht-core` directory.
```

Comment:
```suggestion
2. When you want to check for updates, just run `git pull orign` in the `cht-core` directory.
```

I think this is technically unnecessary since usually markdown rendering engines are smart enough to make the numbering sequential, but can't hurt.

Reply:
thanks!